storage/src/test/java/tech/pegasys/teku/storage/server/AbstractDatabaseTest.java

```diff
@@ -645,6 +647,7 @@ protected void testShouldRecordFinalizedBlocksAndStates(
     assertBlocksFinalized(expectedFinalizedBlocks);
     assertGetLatestFinalizedRootAtSlotReturnsFinalizedBlocks(expectedFinalizedBlocks);
     assertBlocksAvailableByRoot(expectedFinalizedBlocks);
+    assertFinalizedBlocksAvailableViaStream(primaryChain);
```

Comment:
This helper method is oddly specific - why not pass in `expectedFinalizedBlocks`?